components/scream/scripts/test_all_scream.py

```diff
@@ -155,6 +155,8 @@ def test_factory(user_req_tests, machine, mem_check):
     if mem_check:
         for test in result:
             test.longname += "_cuda_mem_check" if is_cuda_machine(machine) else "_valgrind"
+            test.uses_baselines = False
```

Comment:
Is this needed? I see around line 418 that baselines are already disabled if mem_check is on.

Reply:
I thought I was getting an error due to missing baselines on my machine. Perhaps the checks before line 418? Let me verify.

Reply:
You're right, the line was not needed. I'm not sure why I thought I had to add it. I purged it.